rllib/connectors/util.py

```diff
@@ -34,14 +34,20 @@ def get_agent_connectors_from_config(
 ) -> AgentConnectorPipeline:
     connectors = []
 
-    if config["clip_rewards"] is True:
+    # In addition to when specified, we turn on reward clipping by default
+    # for Atari environments.
+    if config["clip_rewards"] is True or ctx.is_atari:
         connectors.append(ClipRewardAgentConnector(ctx, sign=True))
     elif type(config["clip_rewards"]) == float:
         connectors.append(
             ClipRewardAgentConnector(ctx, limit=abs(config["clip_rewards"]))
         )
 
-    if not config["_disable_preprocessor_api"]:
+    if (
+        ctx.preprocessing_enabled
```

Comment:
This flag is also confusing to me. Why would we ever have `processing_enabled=True` and `_disable_preprocessor_api = True`?